apps/lib/s_cb.c

```diff
@@ -1224,7 +1224,7 @@ void print_ssl_summary(SSL *s)
     c = SSL_get_current_cipher(s);
     BIO_printf(bio_err, "Ciphersuite: %s\n", SSL_CIPHER_get_name(c));
     do_print_sigalgs(bio_err, s, 0);
-    peer = SSL_get_peer_certificate(s);
+    peer = SSL_get1_peer_certificate(s);
```

Comment:
As the reference count is increased, can this change cause a memory leak?

Reply:
No, `SSL_get_peer_certificate()` is (was) the same functionality of `SSL_get1_peer_certificate()`. The same applies to all other comments.
See the original code: https://github.com/openssl/openssl/pull/8730/files#diff-ff196ec6869f7b41a0b414a42d68a444L1546

Reply:
Got it. Thanks!